ground_truth_labeling_jobs/ground_truth_conversion_scripts/GT_semantic_segmentation_to_COCO.ipynb

```diff
@@ -0,0 +1,443 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%matplotlib inline\n",
+    "from pycocotools.coco import COCO\n",
+    "import numpy as np\n",
+    "import skimage.io as io\n",
+    "import matplotlib.pyplot as plt\n",
+    "import pylab\n",
+    "pylab.rcParams['figure.figsize'] = (20.0, 10.0)\n",
+    "\n",
+    "annFile='COCO_file.json'\n",
+    "coco=COCO(annFile)\n",
+    "# display COCO categories and supercategories\n",
+    "cats = coco.loadCats(coco.getCatIds())\n",
+    "nms=[cat['name'] for cat in cats]\n",
+    "print('COCO categories: \\n{}\\n'.format(' '.join(nms)))\n",
+    "\n",
+    "nms = set([cat['supercategory'] for cat in cats])\n",
+    "print('COCO supercategories: \\n{}'.format(' '.join(nms)))\n",
+    "\n",
+    "\n",
+    "catIds = coco.getCatIds(catNms=['bike','motorbike']);\n",
+    "imgIds = coco.getImgIds(catIds=catIds );\n",
+    "imgIds = coco.getImgIds(imgIds = [1]) #<<<<<<<<<<<< Change the number for different test images\n",
+    "img = coco.loadImgs(imgIds[np.random.randint(0,len(imgIds))])[0]\n",
+    "print(img)\n",
+    "\n",
+    "I = io.imread(img['file_name'])\n",
+    "h, w, c = I.shape\n",
+    "print(h)\n",
+    "print(w)\n",
+    "plt.axis('off')\n",
+    "plt.imshow(I)\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Print out the annotations"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "plt.imshow(I); plt.axis('off')\n",
+    "annIds = coco.getAnnIds(imgIds=img['id'], catIds=catIds, iscrowd=None)\n",
+    "anns = coco.loadAnns(annIds)\n",
+    "coco.showAnns(anns)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Print and show the segmentation using COCO format"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pycocotools.coco as coco\n",
+    "from pycocotools.coco import COCO\n",
+    "import cv2\n",
+    "\n",
+    "for annotation in annotations:\n",
+    "    with open('COCO_file.json', 'w') as json_file:\n",
+    "        json.dump(annotation, json_file)\n",
+    "    coco=COCO('COCO_file.json')\n",
+    "    I = io.imread('<sample_image>')\n",
+    "    plt.imshow(I)\n",
+    "    coco.showAnns([annotation])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from skimage import io\n",
+    "from skimage.viewer import ImageViewer\n",
+    "import matplotlib.pyplot as plt\n",
+    "from skimage.color import rgb2gray\n",
+    "\n",
+    "\n",
+    "a = io.imread(base64.decodebytes(img_data), plugin='imageio')\n",
+    "grayscale = rgb2gray(a)\n",
+    "grayscale[grayscale > 0] = 1\n",
+    "plt.imshow(a)\n",
+    "plt.show()"
```

Comment:
Is this cell supposed to be here still? img_data isn't defined, and I'm not sure what's supposed to be going on.

Reply:
Fixed this one. This is supposed to pick a random image from the dataset (from the output manifest) then overlay the annotation on the image